gaitmesh_ros/src/annotate_cloud_gaits.cpp

```diff
@@ -49,7 +50,8 @@ int main(int argc, char* argv[])
     nodeHandle.param("mesh_labels_from_texture", mesh_labels_from_texture, false);
     nodeHandle.param("radius_normals", radius_normals, 0.20);
     nodeHandle.param("radius_robot", radius_robot, 0.60);
-    nodeHandle.param("curvature_threshold", curvature_threshold, 0.06);
+    nodeHandle.param("curvature_trot_threshold", curvature_trot_threshold, 0.04);
```

Comment:
NB: Default changed from 0.06 to 0.04. Ok if you found it experimentally but we should note it (done here now) that this is a change in behaviour.